### PR TITLE
Add "release-variant" command

### PR DESCRIPTION
**Pull request by xiangge**
Add "release-variant" command.
As the patch needs support 'dest', so also include the below patch which is the same with Jira PDC-1913.
Will commit this 'dest' patch first, after that, will do the rebase work when merge this mainly patch.
plugin_helpers: Allow overriding 'dest' option.

JIRA: PDC-1913

### CODE REVIEW

File: pdc_client/plugins/release_variant.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+        # CRUD: UPDATE
+        update_parser = self.add_action('update', help='update an existing release_variant')
+        update_parser.add_argument('release')
+        update_parser.add_argument('uid')
+        self.add_release_variant_arguments(update_parser)
+        update_parser.set_defaults(func=self.release_variant_update)
+
+        # CRUD: DELETE
+        # not implemented
```
Comment (hluk) [inline pdc_client/plugins/release_variant.py]: Old comment? Looks implemented now.

**Review by hluk — COMMENTED** [review]
+1

File: pdc_client/plugins/release_variant.py (hunk 1 above)
Comment (xiangge) [reply pdc_client/plugins/release_variant.py]: Yes, another patch here adds the delete function.
Just remove the comment line from that commit.

**Comment by simozhan** [comment]
:+1:

**Comment by xiangge** [comment]
Close this as patches have been merged.

**Pull request closed without merging**